doc/man7/EVP_MAC-SIPHASH.pod

```diff
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-EVP_MAC-Siphash - The SipHash EVP_MAC implementation
+EVP_MAC-SIPHASH - The SipHash EVP_MAC implementation
```

Comment:
https://github.com/openssl/openssl/blob/89e5aaa1d72058404d3ea06bfaeff5334aba202d/providers/default/defltprov.c#L306

Same reasoning as for Poly1305

Reply:
I restored the old name.  Does nobody care about consistency???? :)